config/config.toml.example

```diff
@@ -462,3 +459,6 @@ tidb_record_plan_in_slow_log = 1
 
 # The maximum permitted number of simultaneous client connections. When the value is 0, the number of connections is unlimited.
 max_connections = 0
+
+# Run ddl worker on this tidb-server.
+tidb_enable_ddl = true
```

Comment:
All the other config options are not prefixed with `tidb_`. I think it should be removed for the naming consistency.

Reply:
I disagree here. The goal of the instance section is to unify the names between sysvars and config. The convention of sysvars is to use tidb_* if it is tidb specific. There is a lot of value in having unified.

Reply:
OK.

Reply:
@CbcWestwolf can we add a test to make sure all [instance] config items match the name of a sysvar? Because it is a new section there should be no orphans.